### ISSUE TITLE
How to use

### ISSUE THREAD

**Issue by jestillore** [Naddiseo/babel-plugin-mjsx#6]
How can I use this plugin?
What is the name of the plugin?
I tried `mjsx`, but always throws this exception: `ReferenceError: Unknown plugin "mjsx" specified in "path\\to\\.babelrc" at 0`.

Below is my `.babelrc`

```
{
    "presets": ["es2015"],
    "plugins": ["mjsx"]
}
```

**Comment by Naddiseo** [comment]
The plugin's name is "babel-plugin-mjsx" (same as the npm package ;-) )

**Comment by Naddiseo** [comment]
I've added a little bit more to the README, let me know if you have any further issues.

**Comment by jestillore** [comment]
Unfortunately, I still got the same exception.
`ReferenceError: Unknown plugin "babel-plugin-mjsx"`

**Comment by jestillore** [comment]
![structure](https://cloud.githubusercontent.com/assets/3094154/11428824/05b468e8-94ab-11e5-912d-e16df06da87e.PNG)
Attached is my file structure

Below is my `.babelrc`

```
{
    "presets": ["es2015"],
    "plugins": ["babel-plugin-mjsx"]
}
```

Below is the stack trace:

```
> babel script.js --out-file bundle.js
ReferenceError: Unknown plugin "babel-plugin-mjsx" specified in "C:\\Users\\jestillore\\codebase\\babel\\.babelrc" at 0, attempted to resolve relative to "C:\\Users\\jestillore\\codebase\\babel"
    at C:\Users\jestillore\AppData\Roaming\npm\node_modules\babel-cli\node_modules\babel-core\lib\transformation\file\options\option-manager.js:193:17
    at Array.map (native)
    at Function.normalisePlugins (C:\Users\jestillore\AppData\Roaming\npm\node_modules\babel-cli\node_modules\babel-core\lib\transformation\file\options\option-manager.js:173:20)
    at OptionManager.mergeOptions (C:\Users\jestillore\AppData\Roaming\npm\node_modules\babel-cli\node_modules\babel-core\lib\transformation\file\options\option-manager.js:271:36)
    at OptionManager.addConfig (C:\Users\cerberus\AppData\Roaming\npm\node_modules\babel-cli\node_modules\babel-core\lib\transformation\file\options\option-manager.js:221:10)
    at OptionManager.findConfigs (C:\Users\cerberus\AppData\Roaming\npm\node_modules\babel-cli\node_modules\babel-core\lib\transformation\file\options\option-manager.js:364:16)
    at OptionManager.init (C:\Users\cerberus\AppData\Roaming\npm\node_modules\babel-cli\node_modules\babel-core\lib\transformation\file\options\option-manager.js:412:12)
    at File.initOptions (C:\Users\cerberus\AppData\Roaming\npm\node_modules\babel-cli\node_modules\babel-core\lib\transformation\file\index.js:190:75)
    at new File (C:\Users\cerberus\AppData\Roaming\npm\node_modules\babel-cli\node_modules\babel-core\lib\transformation\file\index.js:121:22)
    at Pipeline.transform (C:\Users\cerberus\AppData\Roaming\npm\node_modules\babel-cli\node_modules\babel-core\lib\transformation\pipeline.js:42:16)
```

**Comment by Naddiseo** [comment]
Ah my bad. You are [actually supposed](https://github.com/babel/babel/blob/v6.2.2/packages/babel-core/src/transformation/file/options/option-manager.js#L130) to use "mjsx" for the plugin name. Babel will add in the `babel-plugin-` part itself. I haven't actually upgraded my local development to use babel6 yet, so I haven't tested this plugin as an actual plugin yet.

Anyway, that still will cause error because the npm package doesn't contain the "lib" folder. 

Should be fixed/working as of 2.0.3

**Comment by jestillore** [comment]
It works now. Thanks.
